internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -550,24 +552,24 @@ func fetchConfig(ctx context.Context, r runtime.Runtime) (out []byte, err error)
 	return b, nil
 }
 
-func receiveConfigViaWebconfig(ctx context.Context, logger *log.Logger, r runtime.Runtime) ([]byte, error) {
-	cfgBytes, err := webconfig.Run(ctx, logger, r.State().Platform().Mode())
+func receiveConfigViaInteractiveInstaller(ctx context.Context, logger *log.Logger, r runtime.Runtime) ([]byte, error) {
```

Comment:
Standardized on calling it "interactive installer" ... happy to change, but I don't think it is "webconfig" after this change.